docker/run_crontabber.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+# Runs crontabber.
+
+# Run crontabber sleeping 5 minutes between runs
+while true; do
+    python socorro/cron/crontabber_app.py
+    echo "Sleep 5 minutes..."
+    sleep 300
+done
```

Comment:
+1 on this
crontabber is smart to know to not run any jobs if it's already in the middle of running something. That's how it determines to exit with 0 or 2 (or 3 in extreme cases). 
So there's no "they'll fight to the death and then things keep happening" :)

I had to check but if any exception happens *within* a crontabber app, it will exit with 0 nevertheless.
See https://github.com/mozilla/crontabber/blob/57c463a521cdd2f9db4ae3c1104559abf88d1607/crontabber/app.py#L840-L852

We should put `set -e` at the top of this file actually. Then if something is wrong with the configuration (e.g. a typo in `crontabber.env` like `jobs=socorro.cron.jobs.bogzilla.BogzillaCronApp`) then it won't just sit there failing start at all over and over.